js/mobile-menu.js

```diff
@@ -0,0 +1,54 @@
+const desktopFooter = document.querySelector('.desktop-footer');
+const contactForm = document.querySelector('.contact-form');
+const hamburgerBtn = document.getElementById('hamburger-button');
+const mobileMenu = document.getElementById('mobile-menu');
+const desktopVersion = document.querySelector('.main1');
+const mobileVersion = document.querySelector('.main');
+
+function handleVisibility() {
+  const screenWidth = window.innerWidth || document.documentElement.clientWidth;
+
+  if (screenWidth >= 768) {
+    // Show desktop footer and hide contact form
+    desktopFooter.style.display = 'block';
+    contactForm.style.display = 'none';
+  } else {
+    // Show contact form and hide desktop footer
+    desktopFooter.style.display = 'none';
+    contactForm.style.display = 'block';
+  }
+
+  if (screenWidth >= 768) {
+    // Show desktop version and mobile version
+    desktopVersion.style.display = 'block';
+    mobileVersion.style.display = 'none';
+  } else {
+    // Show mobile version and hide desktop version
+    desktopVersion.style.display = 'none';
+    mobileVersion.style.display = 'block';
+  }
```

Comment:
why repeat the same conditional statment... why not move the logics in one conditional statement?